src/components/Editor/Editor.data.tsx

```diff
@@ -30,8 +30,12 @@ foo1();`,
 	{
 		title: 'microtasks',
 		code: `console.log(1);  
-setTimeout(() => console.log(2), 0);  
-queueMicrotask(() => console.log(3));  
+console.log(2-1);
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously. -->
🎯 suggestion:
Adding an additional 5 binary expression examples to the "microtasks" example may create confusion as they are not related to microtasks. 
What do you think about adding some of them to "everything" instead?